examples/scale/scenarios/first_match_scenario.go

```diff
@@ -0,0 +1,98 @@
+package scenarios
+
+import (
+	"fmt"
+	"io"
+	"open-match.dev/open-match/pkg/pb"
+	"time"
+)
+
+var (
+	firstMatchScenario = &Scenario{
+		MMF:                          queryPoolsWrapper(firstMatchMmf),
+		Evaluator:                    fifoEvaluate,
+		FrontendTotalTicketsToCreate: -1,
+		FrontendTicketCreatedQPS:     100,
+		BackendAssignsTickets:        true,
+		BackendDeletesTickets:        true,
+		Ticket:                       firstMatchTicket,
+		Profiles:                     firstMatchProfile,
+	}
+)
+
+func firstMatchProfile() []*pb.MatchProfile {
+	return []*pb.MatchProfile{
+		{
+			Name: "entirePool",
+			Pools: []*pb.Pool{
+				{
+					Name: "all",
```

Comment:
can you use a variable for `all` instead? you also refer to this literal at line 41 and this correlation is not obvious at first glance.